src/views/ModelPlan/TaskList/Basics/Overview/index.tsx

```diff
@@ -0,0 +1,337 @@
+                    </FieldGroup>
+
+                    <FieldGroup
+                      scrollElement="problem"
+                      error={!!flatErrors.problem}
+                      className="margin-top-4"
+                    >
+                      <Field
+                        as={TextAreaField}
+                        className="height-15"
+                        error={flatErrors.problem}
+                        id="ModelType-Problem"
+                        name="problem"
+                        label={t('problem')}
+                      />
+                    </FieldGroup>
+
+                    <FieldGroup
+                      scrollElement="goal"
+                      error={!!flatErrors.goal}
+                      className="margin-top-4"
+                    >
+                      <Field
+                        as={TextAreaField}
+                        className="height-15"
+                        error={flatErrors.goal}
+                        id="ModelType-Goal"
+                        name="goal"
+                        hint={t('goalHelp')}
+                        label={t('goal')}
+                      />
+                    </FieldGroup>
+
+                    <FieldGroup
+                      scrollElement="testInterventions"
+                      error={!!flatErrors.testInterventions}
+                      className="margin-top-4"
+                    >
+                      <Field
+                        as={TextAreaField}
+                        className="height-15"
+                        error={flatErrors.testInterventions}
+                        id="ModelType-testInterventions"
+                        name="testInterventions"
+                        label={t('testInterventions')}
+                      />
+                    </FieldGroup>
+
+                    <Button
+                      type="button"
+                      className="usa-button usa-button--unstyled margin-top-4"
+                      onClick={() => setHasAdditionalNote(true)}
+                    >
+                      <IconAdd className="margin-right-1" aria-hidden />
+                      {h('additionalNote')}
+                    </Button>
+
+                    {hasAdditionalNote && (
+                      <FieldGroup className="margin-top-4">
+                        <Field
+                          as={TextAreaField}
+                          className="height-15"
+                          id="ModelType-note"
+                          name="note"
+                          label={t('Notes')}
+                        />
+                      </FieldGroup>
+                    )}
+
+                    <div className="margin-top-6 margin-bottom-3">
+                      <Button
+                        type="button"
+                        className="usa-button usa-button--outline margin-bottom-1"
+                        onClick={() => history.goBack()}
```

Comment:
According to UX, we rarely want to use the browser's back functionality and instead want to always route to the previous form page.  In the previous comment, there was another option for 'back' as a route prop for form submission that can be used to explicitly set the previous route.